### DISCUSSION TITLE
Release of 7.4-SNAPSHOT

### DISCUSSION THREAD

**Discussion by hugithordarson** [wocommunity/wonder#996] [General]
Should we start working towards making a release of the current master branch? Apart from everything else, just the work by @nullterminated in #994 is significant and I'm willing to put in some work to get a clean release going :).

**Comment by paulhoadley** [comment]
> Should we start working towards making a release of the current master branch?

Yes.

**Comment by nullterminated** [comment]
I can help do this, but I'll need help restoring my credentials on the build server. It has been a long time since I set up a wocommunity build.

**Reply by hugithordarson** [reply]
At some point we were throwing around the idea to migrate our builds from jenkins to github. Perhaps this is an opportunity to make a move? @hprange already made a functioning build action, the fun part would probably be deployment to the WOCommunity maven repo.

**Comment by nullterminated** [comment]
We still need the wocommunity server for the WO artifacts, so we may as well use that. Github could start charging money tomorrow for what is free today. It's nice to have the code here, because it's easier for people to contribute. It seems wiser to stay on the wocommunity server for builds to me. Plus Jenkins is easier, I'm very familiar with Jenkins :) Github yamls not so much.
